integration-tests/src/test/java/oracle/weblogic/kubernetes/ItStickySession.java

```diff
@@ -537,24 +536,24 @@ private void sendHttpRequestsToTestSessionStickinessAndVerify(String hostname, i
 
     // send a HTTP request again to get server and session info
     httpDataInfo =
-      getServerAndSessionInfoAndVerify(hostname,
-        ingressServiceNodePort, webServiceGetUrl, " -b ");
+        getServerAndSessionInfoAndVerify(hostname,
+            ingressServiceNodePort, webServiceGetUrl, " -b ");
     // get server and session info from web service deployed on the cluster
     String serverName2 = httpDataInfo.get(serverNameAttr);
     String sessionId2 = httpDataInfo.get(sessionIdAttr);
     String countStr = httpDataInfo.get(countAttr);
-    int count = Optional.ofNullable(countStr).map(Ints::tryParse).orElse(0);
+    int count = Optional.ofNullable(countStr).map(Integer::valueOf).orElse(0);
     logger.info("Got the server {0}, session ID {1} and session state {2} "
```

Comment:
Ints.tryParse() doesn't throw exception if the string is not a valid integer, but Integer.valueOf can throw NumberFormatException. Would that be an issue here?

Reply:
I don't think that it would be an issue as we would just see the test fail.